exchange/exchangetest/aliases.json

```diff
@@ -58,7 +58,13 @@
             }
           ],
           "ext": {
-            "prebid": {}
+            "prebid": {
+              "server": {
+                "datacenter": "Datacenter",
+                "externalurl": "http://hosturl.com",
+                "gvlid": 1
+            }
+            }
```

Comment:
Nitpick: Looks good, just please run a formatter on the updates json files. There shouldn't be too closing braces at the same indentation level. There are several json files with this, just likely from copying and pasting the new server object.